Lara-JS/src-api/lara/util/DataStore.ts

```diff
@@ -0,0 +1,270 @@
+import Io from "../Io.js";
+import { info } from "../core/LaraCore.js";
+import JavaTypes, { JavaClasses } from "./JavaTypes.js";
+
+/**
+ * Interface with Java class DataStore, used for storing arbitrary information.
+ */
+
+export default class DataStore {
+  javaDataStoreInstance!: JavaClasses.DataStore;
+  definition: any;
+  checkKeys: boolean;
+  allowedKeys: any;
```

Comment:
Set<string> | undefined

Reply:
I assume this comment refers to the `allowedKeys` field